dvc/pkg/config.py

```diff
@@ -0,0 +1,35 @@
+from dvc.config import ConfigError
+
+
+def get_pkg_settings(config, name):
+    settings = config.config.get(config.SECTION_PKG_FMT.format(name))
+
+    if settings is None:
+        raise ConfigError("unable to find package section '{}'".format(name))
+
+    return settings
+
+
+def pkg_add(config, name, url, level=None):
+    return config.set(
+        config.SECTION_PKG_FMT.format(name),
+        config.SECTION_PKG_URL,
```

Comment:
the same here? why don't we move PKG specific stuff out of config?

Reply:
https://github.com/iterative/dvc/pull/2012#discussion_r290230526